loafwalletTests/Two Factor Auth Tests/Enter2FACodeViewModelTests.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  TwoFactorAuthTests.swift
+//  loafwalletTests
+//
+//  Created by Kerry Washington on 8/15/21.
+//  Copyright © 2021 Litecoin Foundation. All rights reserved.
+//
+
+import XCTest
+@testable import loafwallet
+
+class Enter2FACodeViewModelTests: XCTestCase {
+    
+    var viewModel: Enter2FACodeViewModel!
+
+    override func setUp() {
+        super.setUp()
+        viewModel = Enter2FACodeViewModel()
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    func testExample() throws {
```

Comment:
remove dead funcs (testExampe() and testPerformanceExample())

Reply:
I'll remove. Tx.